server/src/main/java/org/elasticsearch/index/query/MatchPhraseQueryBuilder.java

```diff
@@ -67,6 +71,7 @@ public MatchPhraseQueryBuilder(StreamInput in) throws IOException {
         fieldName = in.readString();
         value = in.readGenericValue();
         slop = in.readVInt();
+        zeroTermsQuery = ZeroTermsQuery.readFromStream(in);
```

Comment:
You'll need to handle backward compatibility here. You can check https://github.com/elastic/elasticsearch/blob/5263b8cc7ee7b1a52880ab98587b88d6c00557b8/server/src/main/java/org/elasticsearch/index/query/MatchQueryBuilder.java#L132 for an example (the version should be v7_0...).